doc/classes/Node.xml

```diff
@@ -982,6 +992,9 @@
 		<member name="editor_description" type="String" setter="set_editor_description" getter="get_editor_description" default="&quot;&quot;">
 			An optional description to the node. It will be displayed as a tooltip when hovering over the node in the editor's Scene dock.
 		</member>
+		<member name="exposed_in_owner" type="bool" setter="set_exposed_in_owner" getter="is_exposed_in_owner" default="false">
+			Sets this node's state as an exposed node in its [member owner]. This allows the node to be accessed when the scene is instantiated in another. It also enforces the node to become unique, see [member unique_name_in_owner].
```

Comment:
The description could use more clarification. It should mention that it's similar to editable children.

> This allows the node to be accessed when the scene is instantiated in another.

This applies to any node, the important difference here is that the node appears in the editor.